src/components/auth/index.js

```diff
@@ -0,0 +1,4 @@
+export * from './ForgotPassword'
```

Comment:
Added a barrel to export everything from within the auth folder, considering that we will probably use these everywhere in our app

Reply:
I didn't know you could export all named functions using *. Good lesson